bucket/snaptimer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.1",
+    "description": "Windows Countdown Timer",
+    "homepage": "https://dan.hersam.com/software/snaptimer/index.php",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://dan.hersam.com/software/license.php"
+    },
+    "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+    "hash": "sha1:7131F4C79038B8059F8EF16601077F075CFAD2DC",
```

Comment:
```suggestion
    "hash": "sha1:7131f4c79038b8059f8ef16601077f075cfad2dc",
```

Reply:
1. Have read [Allowing changes to a pull request branch created from a fork - GitHub Docs](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork), but there are no such options, maybe needs to turn on when first-time sending pull request.

![1604454911](https://user-images.githubusercontent.com/16096559/98060220-0e568680-1e84-11eb-8af0-7f5435980124.png)
2. sorry for having uppercased hash
3. I am afraid there is no version 0.2, the software are develop 10 years ago(https://github.com/jaden/SnapTimer).
   - https://dan.hersam.com/map.php
   - https://dan.hersam.com/software/snaptimer/index.php

   so should we omit that version 0.2? :|